index.js

```diff
@@ -1,5 +1,8 @@
+// TODO: add rule for `.restore` being called on methods, suggest it gets called on sinon or sandbox only
+
 module.exports = {
   rules: {
+    'stubbing-in-nested-before-each': require('./rules/stubbingInNestedBeforeEach'),
```

Comment:
Maybe `no-stubbing-in-nested-before-each`?
https://eslint.org/docs/developer-guide/working-with-rules#rule-naming-conventions